sunpy/map/tests/test_mapbase.py

```diff
@@ -1058,20 +1058,15 @@ def test_resample_rotated_map_cd(cd, method, simple_map):
     smap.meta.update(cd)
     for key in ['cdelt1', 'cdelt2', 'pc1_1', 'pc1_2', 'pc2_1', 'pc2_2']:
         del smap.meta[key]
-    # Check superpixel with a rotated map with unequal resampling
-    new_dims = (1, 2) * u.pix
+    # Check rebin with a rotated map with unequal resampling
+    new_dims = (3, 1) * u.pix
     new_map = getattr(smap, method)(new_dims)
     # Coordinate of the lower left corner should not change
     ll_pix = [-0.5, -0.5]*u.pix
     assert smap.wcs.pixel_to_world(*ll_pix).separation(
         new_map.wcs.pixel_to_world(*ll_pix)).to(u.arcsec) < 1e-8 * u.arcsec
```

Comment:
I had to change this to match for the new 9 by 9 but now the assert fails with a much larger separation angle.